skins/laika/src/components/pages/membership_form/MembershipType.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <fieldset class="has-margin-top-36">
+        <legend class="title is-size-5">{{ $t('membership_membershiptype_legend') }}</legend>
+        <div>
+            <b-radio :class="{ 'is-active': selectedType === 'sustaining' }"
+                    type="radio"
+                    id="sustaining"
+                    name="type"
+                    v-model="selectedType"
+                    native-value="sustaining"
+                    @change.native="setType">
+                <span>{{ $t( 'membership_membershiptype_option_sustaining' ) }}</span>
+                <p>{{ $t( 'membership_membershiptype_option_sustaining_legend' ) }}</p>
+            </b-radio>
+            <b-radio :class="{ 'is-active': selectedType === 'active' }"
+                    type="radio"
+                    id="active"
+                    name="type"
+                    v-model="selectedType"
+                    native-value="active"
+                    @change.native="setType">
+                {{ $t( 'membership_membershiptype_option_active' ) }}
+                <p>{{ $t( 'membership_membershiptype_option_active_legend' ) }}</p>
+            </b-radio>
+        </div>
+    </fieldset>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend( {
+	name: 'MembershipType',
+	data: function () {
+		return {
+			selectedType: 'sustaining',
+		};
+	},
+	methods: {
+		setType(): void {
+			// TODO: store the chosen membership type in the membership address store once it exists
+		},
+	},
+} );
+</script>
+<style lang="scss" scoped>
+    .b-radio.radio {
+        margin-left: 0;
+        height: 6.5em;
+    }
+    p {
+        color: rgba(0, 0, 0, 0.6);
```

Comment:
It's better to use the color variables and auto-generated CSS classes for this. You can assign the `has-text-dark-lighter` CSS class to the `<p>` elements and remove this CSS rule, this will apply `$fun-color-dark-lighter` which is the same as yours.